the-blue-alliance-ios/Core Data/Event/EventRankingStatInfo.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import CoreData
+
+extension EventRankingStatInfo: Managed {
+
+    class func insert(_ model: TBAEventRankingSortOrder, in context: NSManagedObjectContext) -> EventRankingStatInfo {
+        let predicate = NSPredicate(format: "%K == %@ && %K == %ld",
```

Comment:
Maybe just query by name here - seems possible we'd update the precision but keep the name upstream